registry/slashFun.js

```diff
@@ -0,0 +1,163 @@
+    new u.int()
+    .setName("dice")
+    .setDescription("How many dice to roll? (will roll at least 1 anyway) (max of 10000)")
+    .setRequired(false)
+    .setMinValue(1)
+    .setMaxValue(10000)
+  )
+  .addIntegerOption(
+    new u.int()
+    .setName("sides")
+    .setDescription("How many sides on the dice to roll? (defaults to 6)")
+    .setRequired(false)
+    .setMinValue(1)
+    .setMaxValue(2147483647)
+  )
+  .addIntegerOption(
+    new u.int()
+    .setName("modifier")
+    .setDescription("How much to change the roll by? (defaults to 0)")
+    .setRequired(false)
+  );
+const rollF = new u.sub()
+  .setName("rollf")
+  .setDescription("Roll Fate Dice")
+  .addIntegerOption(
+    new u.int()
+    .setName("dice")
+    .setDescription("How many dice to roll? (will roll at least 1 anyway) (max of 10000)")
+    .setRequired(false)
+    .setMinValue(1)
+    .setMaxValue(10000)
+  )
+  .addIntegerOption(
+    new u.int()
+    .setName("modifier")
+    .setDescription("How much to change the roll by? (defaults to 0)")
+    .setRequired(false)
+  );
+const rollOld = new u.sub()
+  .setName("rollold")
+  .setDescription("Roll Dice using the old formula format")
+  .addStringOption(
+    new u.string()
+    .setName("rollformula")
+    .setDescription("old dice formula to use, it errors in the same way as !roll did")
+  );
+const ball8 = new u.sub()
+  .setName("8ball")
+  .setDescription("Get an answer from the Magic 8-ball.")
+  .addStringOption(
+    new u.string()
+    .setName("question")
+    .setDescription("What do you wish to ask the 8-ball today?")
+    .setRequired(true)
+  );
+const allthe = new u.sub()
+  .setName("allthe")
+  .setDescription("ALL THE _____!")
+  .addStringOption(
+    new u.string()
+    .setName("thing")
+    .setDescription("something")
+    .setRequired(true)
+  );
+const repost = new u.sub()
+.setName("repost")
+.setDescription("That's a repost.");
+const acronym = new u.sub()
+.setName("acronym")
+.setDescription("Get a random 3-5 letter acronym. For science.");
+const mines = new u.sub()
+  .setName("mines")
+  .setDescription("Play a game of Minesweeper!")
+  .addStringOption(
+    new u.string()
+    .setName("difficulty")
+    .setDescription("5 by 5 with 5 mines, 10 by 10 with 30 mines, or 14 by 14 with 60 mines")
+    .setRequired(true)
+    .setChoices(
+      { name: "Easy", value: "Easy" },
+      { name: "Medium", value: "Medium" },
+      { name: "Hard", value: "Hard" })
+  );
+const hbs = new u.sub()
+  .setName("hbs")
+  .setDescription("Play a game of Handicorn, Buttermelon, Sloth!")
+  .addStringOption(
+    new u.string()
+    .setName("choice")
+    .setDescription("your choice of Handicorn, Buttermelon, or Sloth!")
+    .setRequired(true)
+    .setChoices(
+      { name: "Handicorn", value: "Handicorn" },
+      { name: "Buttermelon", value: "Buttermelon" },
+      { name: "Sloth", value: "Sloth" })
+  )
+  .addStringOption(
+    new u.string()
+    .setName("vsmode")
```

Comment:
either add a - or change it to `mode` (preferred)

Reply:
I tried it as mode and it sent weird errors, and all I changed was the name of the parameter to vsmode and they went away.

Reply:
I'll try again.

Reply:
of course it works now.